src/ol/format/WMSGetFeatureInfo.js

```diff
@@ -111,31 +104,43 @@ class WMSGetFeatureInfo extends XMLFeature {
           continue;
         }
 
-        const featureType = layerName +
-            featureIdentifier;
+        const featureType = layerName + featureIdentifier;
 
         context['featureType'] = featureType;
         context['featureNS'] = this.featureNS_;
 
         /** @type {Object<string, import("../xml.js").Parser>} */
         const parsers = {};
         parsers[featureType] = makeArrayPusher(
-          this.gmlFormat_.readFeatureElement, this.gmlFormat_);
+          this.gmlFormat_.readFeatureElement,
+          this.gmlFormat_
+        );
         const parsersNS = makeStructureNS(
-          [context['featureNS'], null], parsers);
+          [context['featureNS'], null],
+          parsers
+        );
         layerElement.setAttribute('namespaceURI', this.featureNS_);
         const layerFeatures = pushParseAndPop(
           // @ts-ignore
```

Comment:
It [looks like](https://app.circleci.com/pipelines/github/openlayers/openlayers/3491/workflows/6e8f1da1-bc7e-40e2-a13b-4f41d0f33b2a/jobs/3566) this comment needs to get pushed down a couple lines.  I'll update in a new commit.

Reply:
Addressed in ad851e779f4545bc0a156820e0943f9a7bca0dbc.